File: Programmers/Level1/소수 만들기.py
def solution(nums):
    answer = 0
    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            for k in range(j+1, len(nums)):
                flag = False
                number = nums[i] + nums[j] + nums[k]
                for l in range(2, number):
                    if(number % l == 0):
                        flag = True
                        break
                if(not flag):
                    answer += 1

    return answer

File: Swift 코딩테스트 연습/소수 만들기.swift
import Foundation

func solution(_ nums:[Int]) -> Int {
    var answer = 0
    var sums: [Int] = []
    
    for i in 0..<nums.count {
        for j in i+1..<nums.count {
            for k in j+1..<nums.count {
                sums.append(nums[i] + nums[j] + nums[k])
            }
        }
    }
    
    for i in 0..<sums.count {
        var isPrimeNum:Bool = true
        for j in 2..<Int(sqrt(Double(sums[i])) + 1) {
            if sums[i] % j == 0 {
                isPrimeNum = false
                break
            }
        }
        if isPrimeNum {
            answer += 1
        }
    }
    
    return answer
}